db/migrate/20240513144147_create_thresholds_set.rb

```diff
@@ -0,0 +1,17 @@
+class CreateThresholdsSet < ActiveRecord::Migration[6.1]
+  def change
+    create_table :threshold_sets do |t|
+      t.string :sensor_name
+      t.string :unit_symbol
+      t.boolean :is_default
+      t.float :threshold_very_low
+      t.float :threshold_low
+      t.float :threshold_medium
+      t.float :threshold_high
+      t.float :threshold_very_high
```

Comment:
A wondering:
On the frontend we have min, low, middle, high, max. Is it something we'd like to have across the whole codebase?